repo-projects/Directory.Build.targets

```diff
@@ -265,10 +265,11 @@
     </Touch>
   </Target>
 
-  <!-- TODO: Remove when all repos use a consistent set of eng/common files: https://github.com/dotnet/source-build/issues/3710. -->
-  <Target Name="UpdateEngCommonFiles" Condition="'$(UseBootstrapArcade)' != 'true'">
+  <!-- Don't update arcade's eng/common layout as that is considered live even though it uses bootstrap Arcade. -->
+  <Target Name="UpdateEngCommonFiles" Condition="'$(RepositoryName)' != 'arcade'">
     <ItemGroup>
-      <InputEngCommonFile Include="$(RepoRoot)src\arcade\eng\common\**\*" />
+      <InputEngCommonFile Include="$(RepoRoot)eng\common\**\*" Condition="'$(UseBootstrapArcade)' == 'true'" />
```

Comment:
This is really not the right version either.  The only correct way to solve this is to capture this within PSB and reference it from there.

Reply:
The eng/common layout from the VMR root is the one that must be in sync with bootstrap Arcade from PSB. We need to guarantee that that's the case.

Reply:
Thanks a lot